modules/ossm-uguide-routing-ingress.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_user_guide/ossm-traffic-manage.adoc
+
+[id="ossm-routing-ingress_{context}"]
+= Managing ingress traffic
+
+In a service mesh, you can use a {ProductName} gateway to enable {ProductShortName} features such as monitoring and route rules to be applied to traffic entering the cluster.
```

Comment:
Suggestion
In Red Hat OpenShift Service Mesh, you use the Ingress Gateway to enable Service Mesh features such as monitoring, security, and route rules to be applied to traffic entering the cluster.  Traffic coming into the mesh must come in via the Ingress Gateway.
You configure Service Mesh to expose a service outside of the service mesh using an Service Mesh gateway.

Reply:
>Traffic coming into the mesh must come in via the Ingress Gateway.

I think this is true, but I remember now talking to Rob. Traffic does come into the mesh through the Ingress gateway, but it does that by default. Configuring the service outside the service mesh is a non-default special case and that's why it's lower down. It's only used by a subset of users.

I'm making the second paragraph part of the first to make that more clear.